packages/mux-uploader/src/constants.ts

```diff
@@ -0,0 +1 @@
+export const PROGRESS_INDICATOR_TYPE_BAR = 'bar';
```

Comment:
suggestion(non-blocking): for "enum-like" and other related-constant lookups, we'll typically do something like:
```ts
export type ProgressIndicatorTypes = {
  BAR: 'bar';
  RADIAL: 'radial';
};

export const ProgressIndicatorTypes: ProgressIndicatorTypes = {
  BAR: 'bar',
  RADIAL: 'radial',
};

```